### PR TITLE
docs: some additional v3 FAQ entries.

### PR DESCRIPTION
**Pull request by htuch**
* Elaborate on what we gained from v3.

* Explain how to rollout control plane changes.

Signed-off-by: Harvey Tuch <htuch@google.com>

### CODE REVIEW

File: docs/root/faq/api/why_versioning.rst [hunk 1]
```diff
@@ -12,3 +12,24 @@ We had previously put in place policies around :repo:`breaking changes
 <api/API_VERSIONING.md>` takes this a step further, articulating a guaranteed multi-year support
 window for APIs that provides control plane authors a predictable clock when considering support
 for a range of Envoy versions.
+
+For the v3 xDS APIs, a brief list of the key improvements that were made with a clean break from v2:
```
Comment (htuch) [inline docs/root/faq/api/why_versioning.rst:16]: @markdroth FYI

File: docs/root/faq/api/why_versioning.rst [hunk 2]
```diff
@@ -12,3 +12,24 @@ We had previously put in place policies around :repo:`breaking changes
...
+
+* Packages organization was improved to reflect a more logical grouping of related APIs:
```
Comment (htuch) [inline docs/root/faq/api/why_versioning.rst:18]: @envoyproxy/api-shepherds please LMK if I've missed any major v3 win here. Thanks!

Comment (htuch) [reply docs/root/faq/api/why_versioning.rst:18]: @kyessenov as well

File: docs/root/faq/api/why_versioning.rst (hunk 1 above)
Comment (markdroth) [reply docs/root/faq/api/why_versioning.rst:16]: LGTM.  Thanks for the info!

**Pull request merged**